src/cmd-shell

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env bash
+
+if [[ -z "$@" ]]; then
+    # If no arguments then offer user bash prompt inside the container
+    exec /usr/bin/env PS1='[coreos-assembler]$ ' /usr/bin/bash
```

Comment:
Not worth respinning right now but I think it'd be a bit better to change the default bash prompt in the container.

(Really something we should make saner in our base container image)

Reply:
i.e. populate a `.bashrc` in the container itself?

Reply:
Yeah.